packages/docs-site/src/content/docs/docs/platforms/netlify.mdx

```diff
@@ -0,0 +1,162 @@
+
+
+## Migrating from Netlify managed config
+
+
+**Should I set env vars in the Netlify UI at all?**
+
+Of course you _can_ still set overrides and secrets within the Netlify UI if you like and rely on config values being injected that way.
+
+However, we recommend migrating all of your config to DMNO itself by using [switchBy](/docs/reference/helper-methods/#switchby) to create branching logic based on the env vars injected by Netlify.
+
+You can also use plugins - for example our [Encrypted Vault](/docs/plugins/encrypted-vault/) and [1Password](/docs/plugins/1password/) plugins - to handle sensitive config within your schema. Note that you will still set a **single environment variable** in the netlify UI, to allow these plugins to access the rest of your secret.
+
+For example
+```ts title='.dmno/config.mts'
+import { defineDmnoService, switchBy, pickFromSchemaObject } from 'dmno';
+import { NetlifyEnvSchema } from '@dmno/netlify-platform/types';
+import {
+  EncryptedVaultDmnoPlugin, EncryptedVaultTypes,
+} from '@dmno/encrypted-vault-plugin';
+
+// you could use a single vault, but it's best practice
+// to split out prod secrets to limit access
+const DevSecretsVault = new EncryptedVaultDmnoPlugin('vault/dev', {
+  key: configPath('DMNO_VAULT_KEY_DEV'),
+  name: 'dev',
+});
+const ProdSecretsVault = new EncryptedVaultDmnoPlugin('vault/prod', {
+  key: configPath('DMNO_VAULT_KEY_PROD'),
+  name: 'prod',
+});
+
+export default defineDmnoService({
+  schema: {
+    DMNO_VAULT_KEY_DEV: { extends: EncryptedVaultTypes.encryptionKey },
+    DMNO_VAULT_KEY_PROD: { extends: EncryptedVaultTypes.encryptionKey },
+    ...pickFromSchemaObject(NetlifyEnvSchema, 'CONTEXT'),
+    APP_ENV: {
+      value: switchBy('CONTEXT', {
+        _default: 'local',
+        'deploy-preview': 'staging',
+        'branch-deploy': 'staging',
+        production: 'production',
+      }),
+    },
+    SOME_API_KEY: {
+      value: switchBy('APP_ENV', {
+        _default: 'not-sensitive-dev-key',
+        staging: DevSecretsVault.item(),
+        production: ProdSecretsVault.item(),
+      }),
+    },
+  },
+});
+```
+
+### Scopes
+
+Note that netlify's built-in environment variable tooling has a concept of ["scopes"](https://docs.netlify.com/environment-variables/overview/#scopes) and certain vars being applied/available at different times, for example during the build versus during function execution. There are also some limitations around env vars being injected into functions due to the nature of how those platforms are set up.
+
+Using DMNO, we avoid all of that since we inject all of your _resolved_ DMNO config at build time, and standardize access to your config regardless of the situation.
+
+Note that any env vars set in the netlify UI or netlify.toml must include the "build" scope to be noticed by DMNO - since we inject your config at build time.
```

Comment:
```suggestion
Note that any env vars set in the Netlify UI or `netlify.toml` must include the `build` scope to be noticed by DMNO - since we inject your config at build time.
```